### DISCUSSION TITLE
공모전 월드 연동 관련해서 문의 드립니다

### DISCUSSION THREAD

**Discussion by yongin0814** [naverz/zepeto-studio-kor#917] [5. Q&A]
안녕하세요 제페토 공모전 월드 맵 제작중 문의할 것이 있어 메일을 보냅니다.
저희 팀 기획 컨셉 상 세개의 씬을 연결하도록 만들었습니다.

월드 하나를 씬으로 생각했을 때,
[월드1 - 포탈 - 월드2 - 포탈 - 월드3]
입니다.

문의 사항:
1) 맵 용량이 500mb로 명시 되는 것이 월드 한개당의 용량인지 궁금합니다.
2) 저희가 제출하였을 때 팀작으로 인정 받을 수 있는지 궁금합니다

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 모든 씬을 포함한 전체 빌드 패키지 용량이 500mb입니다.
접수하셨을때 팀으로 신청하셨다면 팀으로 인정 받으실 수 있습니다.
감사합니다.
